javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,29 @@
+//gif
+const gifForm = document.getElementById("gif-form");
+const gifSearch = document.querySelector("#search-gif");
+const gifBtn = document.querySelector("#search-gif-btn");
+const gifShow = document.querySelector(".show-gif");
+
+gifForm.addEventListener("submit", (event) => {
+  event.preventDefault();
+  console.log("test test test ok");
+  const gifSearch = gifForm.elements.search.value;
+  const gifNumLimit = gifForm.elements.number.value;
+  const toNum = parseInt(gifNumLimit);
+  console.log(toNum);
+  console.log(gifSearch);
```

Comment:
Does not affect the app though, but consider deleting the logs before posting your code somewhere like git. Good practice.